tiled/objects.tsx

```diff
@@ -32,7 +32,7 @@
   </properties>
   <image width="16" height="16" source="images/objects/pressure_plate.png"/>
  </tile>
- <tile id="6">
+ <tile id="6">https://github.com/rmgr/TetraForce.git
```

Comment:
is this a typo?

Reply:
My laptop has this rad feature under debian where if you so much as look at the middle click button too hard it dumps your clipboard where ever you're working. I have no idea why I had objects.tsx open though.. Good catch! I'll update the branch when I get a minute today.

Reply:
Sorted now